message-index/messages/GHC-24180/no-unicode/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Use of ∀ without UnicodeSyntax being enabled
+---
+
+## Message
+```
+Ident.hs:3:10: error: [GHC-24180]
+    Operator applied to too few arguments: ∀
+  |
+3 | ident :: ∀ α.α → α
+  |          ^
+```
+
+## Explanation
+
+The `UnicodeSyntax` language extension causes GHC's parser to recognize a variety of non-ASCII mathematical symbols as representing their Haskell equivalents. The complete list is available in [the documentation](https://downloads.haskell.org/ghc/latest/docs/html/users_guide/exts/unicode_syntax.html).
+
+When this extension is not enabled, `∀` is instead recognized as an infix operator character that has no argument on its left side.
```

Comment:
This seems very strange to be bundled with the above type operator problem, given how much there is a separation of other things by error. But if it is indeed the same error, this is nonblocking on this pr :)

Reply:
This error occurs for users who forget to enable `UnicodeSyntax` and then attempt to write `forall` with `∀`, and I find ti deeply mysterious in that context. I think it's especially useful to document cases where error test is not particularly clearly related to the actual situation.